it/stubs/CustomerInfoStub.scala

```diff
@@ -41,92 +41,74 @@ object CustomerInfoStub extends WireMockMethods {
-      |      "line2":"Bedrock",
-      |      "line3":"Graveldon",
-      |      "line4":"Graveldon",
-      |      "postCode":"GV13 4BJ"
-      |    },
-      |    "contactDetails":{
-      |      "primaryPhoneNumber":"01632 960026",
-      |      "mobileNumber":"07700 900018",
-      |      "emailAddress":"bettylucknexttime@gmail.com"
-      |    }
-      |  }
-      |}""".stripMargin
+  val customerInfoJsonMax: JsValue = Json.obj(
+    "customerDetails" -> Json.obj(
+      "organisationName" -> "Cheapo Clothing Ltd",
+      "firstName" -> "Betty",
+      "lastName" -> "Jones",
+      "tradingName" -> "Cheapo Clothing",
+      "isPartialMigration" ->  false,
+      "vatRegistrationDate" -> "2017-01-01",
+      "customerMigratedToETMPDate" -> "2017-05-05"
+    ),
+    "ppob" -> Json.obj(
+      "address" -> Json.obj(
+        "line1" -> "Bedrock Quarry",
+        "line2" -> "Bedrock",
+        "line3" -> "Graveldon",
+        "postCode" -> "GV2 4BB"
+      ),
+      "contactDetails" -> Json.obj(
+        "primaryPhoneNumber" -> "01632 982028",
+        "mobileNumber" -> "07700 900018",
+        "emailAddress" -> "bettylucknexttime@gmail.com"
+      )
+    ),
+    "bankDetails" -> Json.obj(
+      "accountHolderName" -> "Mrs Betty Jones",
+      "bankAccountNumber" -> "****1234",
+      "sortCode" -> "69****"
+    ),
+    "returnPeriod" -> Json.obj(
+      "stdReturnPeriod" -> "MM"
+    ),
+    "partyType" -> "7",
+    "primaryMainCode" -> "10410"
   )
 
-  val customerInfoHybridUser: JsValue = Json.parse(
-    """{
-      |  "organisationName" : "Cheapo Clothing Ltd",
-      |  "title" : "0001",
-      |  "firstName" : "Betty",
-      |  "middleName" : "Vatreturn",
-      |  "lastName" : "Jones",
-      |  "tradingName" : "Cheapo Clothing",
-      |  "mandationStatus" : "1",
-      |  "registrationReason" : "0001",
-      |  "effectiveRegistrationDate" : "2017-08-21",
-      |  "businessStartDate" : "2017-01-01",
-      |  "isPartialMigration" : true,
-      |  "PPOB":{
-      |    "address":{
-      |      "line1":"Bedrock Quarry",
-      |      "line2":"Bedrock",
-      |      "line3":"Graveldon",
-      |      "line4":"Graveldon",
-      |      "postCode":"GV2 4BB"
-      |    },
-      |    "contactDetails":{
-      |      "primaryPhoneNumber":"01632 982028",
-      |      "mobileNumber":"07700 900018",
-      |      "emailAddress":"bettylucknexttime@gmail.com"
-      |    }
-      |  },
-      |  "correspondenceContactDetails":{
-      |    "address":{
-      |      "line1":"13 Pebble Lane",
-      |      "line2":"Bedrock",
-      |      "line3":"Graveldon",
-      |      "line4":"Graveldon",
-      |      "postCode":"GV13 4BJ"
-      |    },
-      |    "contactDetails":{
-      |      "primaryPhoneNumber":"01632 960026",
-      |      "mobileNumber":"07700 900018",
-      |      "emailAddress":"bettylucknexttime@gmail.com"
-      |    }
-      |  }
-      |}""".stripMargin
+  val customerInfoHybridUser: JsValue = Json.obj(
```

Comment:
Is there a way you can take a copy of customerInfoJsonMax and just update isPartialMigration? Or have a function to generate this data as it's quite hard to distinguish the difference as it it?